cmake/test.cmake

```diff
@@ -221,64 +225,87 @@ finally:
 
     # Emulate what mayaUSD.mod would do
 
-    # adsk
-    list(APPEND MAYAUSD_VARNAME_PATH
-         "${CMAKE_INSTALL_PREFIX}/lib")
-    list(APPEND MAYAUSD_VARNAME_PYTHONPATH
-         "${CMAKE_INSTALL_PREFIX}/lib/scripts")
-    list(APPEND MAYAUSD_VARNAME_MAYA_SCRIPT_PATH
-         "${CMAKE_INSTALL_PREFIX}/lib/scripts")
-    if (IS_LINUX)
-        # On Linux the paths in XBMLANGPATH need a /%B at the end.
-        list(APPEND MAYAUSD_VARNAME_XBMLANGPATH
-             "${CMAKE_INSTALL_PREFIX}/lib/icons/%B")
-    else()
-        list(APPEND MAYAUSD_VARNAME_XBMLANGPATH
-             "${CMAKE_INSTALL_PREFIX}/lib/icons")
+    # MayaUsd core library
+    if (BUILD_MAYAUSD_LIBRARY)
```

Comment:
Moved the various env vars inside of if blocks based on their respective build option.